File: Reflow/Tasks/TaskResult.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Reflow.Tasks
{
    public class TaskResult
    {
        public bool                         IsSuccess           { get; set; }
        public string                       Message             { get; set; }
        public Dictionary<string, object>   Output              { get; set; }
    }
}

File: Java/src/main/java/com/nuarca/etl/tasks/TaskResult.java
package com.nuarca.etl.tasks;

import java.util.HashMap;

public class TaskResult {


    private boolean __IsSuccess;
    public boolean getIsSuccess() {
        return __IsSuccess;
    }

    public void setIsSuccess(boolean value) {
        __IsSuccess = value;
    }

    private String __Message;
    public String getMessage() {
        return __Message;
    }

    public void setMessage(String value) {
        __Message = value;
    }

    private HashMap<String,Object> __Output;
    public HashMap<String,Object> getOutput() {
        return __Output;
    }

    public void setOutput(HashMap<String,Object> value) {
        __Output = value;
    }

}
